content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -283,7 +283,7 @@ curl -X DELETE  'localhost:8080/apis/apps/v1/namespaces/default/replicasets/fron
 
 ### Deleting just a ReplicaSet
 
-You can delete a ReplicaSet without affecting any of its Pods using [`kubectl delete`](/docs/reference/generated/kubectl/kubectl-commands#delete) with the `--cascade=false` option.
+You can delete a ReplicaSet without affecting any of its Pods using [`kubectl delete`](/docs/reference/generated/kubectl/kubectl-commands#delete) with the `--cascade=orphan` option.
```

Comment:
`--cascade=Orphan`

Reply:
I think this value is correct. See the following outputs...

> $ kubectl delete --dry-run=client rs my-replicaset --cascade=false
> warning: --cascade=false is deprecated (boolean value) and can be replaced with --cascade=orphan.

> $ kubectl delete --dry-run=client rs my-replicaset --cascade=Orphan
> error: invalid cascade value (Orphan). Must be "background", "foreground", or "orphan"

Reply:
Okay, thanks for double checking.
Looks like kubectl has performed an additional conversion.